src/util/misc.ts

```diff
@@ -285,3 +285,7 @@ export function updateMap<T1, T2>(
         updatee.set(key, val);
     }
 }
+
+export function last<T>(arr: T[]): T {
```

Comment:
This is not used anywhere. There are several places it can be used. Also assert `arr.length > 0` inside.

Reply:
Fixed in ef446bd.